docs/DeveloperGuide.md

```diff
@@ -350,42 +350,133 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 | `* *`    | ~~tuition teacher~~ | ~~receive notifications when a student's performance drops below a certain threshold~~      | ~~proactively address any learning challenges they might be facing~~                                                 |
 | `*`      | ~~tuition teacher~~ | ~~be able to see a picture of them~~                                                        | ~~easily remember and identify them~~                                                                                |
 
+***
 
 ### Use cases
 
-(For all use cases below, the **System** is the `EduManage` and the **Actor** is the `tuition teacher`, unless specified otherwise)
+(For all use cases below, the **System** is `EduManage` and the **Actor** is the `tuition teacher`, unless specified otherwise)
+
+***
+
+**Use case: Add a student**
+
+**MSS**
+
+1. Tuition teacher requests to add a student by inputting the student's name, phone number, emergency contact and address.
+2. EduManage adds the student's information and indicates success. 
+
+   Use case ends.
+
+**Extensions**
+* 1a. The inputted student's name already exists in EduManage.
+    * 1a1 EduManage shows an error message indicating that the student already exists in EduManage.
+
+      Use case resumes at step 1.
+  
+* 1b. The inputted student's information is missing fields that are not optional.
+  * 1b1 EduManage shows an error message and displays correct command format.
```

Comment:
EduManage shows an error message and informs the user of the correct command format.
Display might be associated to Ui so best to avoid such terms.

Also, for consistency, if you are going to describe the message displayed, would be good to do for all the others.